traitsui/wx/image_enum_editor.py

```diff
@@ -113,13 +117,25 @@ def init(self, parent):
         """ Finishes initializing the editor by creating the underlying toolkit
             widget.
         """
-        self._create_image_grid(parent)
+        super(CustomEditor, self).init(parent)
 
-    def _create_image_grid(self, parent):
+        # Create the panel to hold the ImageControl buttons:
+        self.control = TraitsUIPanel(parent, -1)
+        self._create_image_grid()
+
+    def rebuild_editor(self):
+        # Clear any existing content:
+        self.control.SetSizer(None)
+        GUIToolkit("traitsui", "wx", "traitsui.wx").destroy_children(
```

Comment:
This is already instantiated in `traitsui.wx` and can be imported from `from traitsui.wx import toolkit`.  I see other modules doing this. I am sorry I did not realize this earlier.